src/Compiler.php

```diff
@@ -206,6 +200,7 @@ public function indent(int $step = 1)
      */
     public function outdent(int $step = 1)
     {
+        $this->indentation ??= 0;
```

Comment:
Not necessary, the comparison with null is fine.

Reply:
As it was originally set with step=1, 0 is also set as an error. Should this be explicit?